src/engine/database/gremlin.rs

```diff
@@ -1164,6 +1174,21 @@ impl TryFrom<VertexProperty> for Value {
     }
 }
 
+fn gremlin_comparison_operator(c: &Comparison) -> String {
+    match (&c.operation, &c.negated) {
+        (Operation::EQ, false) => "eq".to_string(),
+        (Operation::EQ, true) => "neq".to_string(),
+        (Operation::CONTAINS, false) => "containing".to_string(),
+        (Operation::CONTAINS, true) => "notContaining".to_string(),
+        (Operation::IN, false) => "within".to_string(),
+        (Operation::IN, true) => "without".to_string(),
+        (Operation::GT, _) => "gt".to_string(),
+        (Operation::GTE, _) => "gte".to_string(),
+        (Operation::LT, _) => "lt".to_string(),
+        (Operation::LTE, _) => "lte".to_string()
+    }
```

Comment:
This was something I had to iterate several times. Originally, every operation was negatable. In cypher every operation is independent and there is the ability to add a `NOT` to the entire clause. When I got to gremlin, they had a distinct operation for negations (eq/neq, within/without, etc) and not for the mathematical operations. In order to maintain consistency, I removed the negated mathematical operations from the `Operation` enum. To your point about what a user might try to do, since we are encoding operations in the schema, users won't be able to enter anything other than what is allowed in the enum.